README.md

```diff
@@ -90,5 +90,5 @@ After that, you should commit updated po files.
 
 4. enable version 1.7 on RTD:
 
-   https://readthedocs.org/dashboard/sphinx-ja/versions/
+   https://readthedocs.org/projects/sphinx/versions/

```

Comment:
@shimizukawa Could you check this please? Surely `/dashboard/sphinx-ja/versions` is NOT FOUND. But I'm not sure `/projects/sphinx/versions` is correct for this case...

Reply:
Recent updates, ReadTheDocs changes URL structure, so `dashboard` is deprecated. 

`/projects/sphinx/versions/` and
`/projects/sphinx-<lang>/versions` are correct URLs.

In this section, as a `sphinx-doc-translations` repository, `-<lang>` is important I think.
So I recommend:
```suggestion
   https://readthedocs.org/projects/sphinx-ja/versions/
```